js/storage.js

```diff
@@ -29,7 +29,27 @@ _Storage.prototype.loadTeams = function() {
 	this.teams = [];
 	if (window.localStorage) {
 		var teamString = localStorage.getItem('showdown_teams');
-		if (teamString) this.teams = JSON.parse(teamString);
+		
+		if (teamString) {
+		    // check if the teams are stored as JSON
+		    // could also try to use JSON.parse and catch errors? (not sure which is faster)
+		    var isJSON = teamString.substring(0,2)>"[{" ? -1 : 0;
```

Comment:
I'm not exactly sure if this will affect anything since I can't actually test it at work, but I think a packed team string could go like `AAA|scyther|eviolite|...`, which causes problems because `("AA" > "[{") === false`. If I'm wrong let me know.